bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/AzimuthListener.java

```diff
@@ -35,7 +35,7 @@ public void propertyChange(PropertyChangeEvent evt) {
 		ApproximateAngle windConstraint = variant.getWindConstraint();
 		if (windConstraint != null) {
 			for (Alloc a: variant.getAllocs()) {
-				AzimuthSolver solver = new AzimuthSolver(schedule.getSite(), a.getObs().getCoords(), windConstraint);
+				AzimuthSolver solver = new AzimuthSolver(schedule.getSite(), a.getObs().getCoords(schedule.getMiddlePoint()), windConstraint);
```

Comment:
Why doesn't `AzimuthSolver` get the same treatment as `ElevationConstraintSolver`s?  This is, why not construct it with a function that takes a time and returns coordinates instead of taking coordinates at a fixed time?